atlasdb-dagger/src/main/java/com/palantir/atlasdb/services/TransactionManagerModule.java

```diff
@@ -143,6 +143,8 @@ public SerializableTransactionManager provideTransactionManager(
                 ConflictTracer.NO_OP,
                 DefaultMetricsFilterEvaluationContext.createDefault(),
                 Optional.empty(),
-                knowledge);
+                knowledge,
+                Executors.newSingleThreadExecutor(new NamedThreadFactory(
```

Comment:
where is this dagger module used? why does the delete executor above use a single thread executor rather than the DefaultTaskExecutors executor?